### PR TITLE
558 pjg monsoon wang parser

### PR DESCRIPTION
**Pull request by gleckler1**
@doutriaux1 Pls review/modify and merge.   This branch involves cleaning up of the monsoon_wang inparam and driver.   Note the example inparam (https://github.com/PCMDI/pcmdi_metrics/blob/558_pjg_monsoon_wang_parser/examples/parameter_files/inparams-example_monsoon-wang.py) there is a pointer to the data to download to run examples.

### CODE REVIEW

File: src/python/monsoon_wang/scripts/mpindex_compute.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
-
 import cdms2
 import numpy
 import sys
 import os
 from genutil import statistics
 from pcmdi_metrics.pcmdi.pmp_parser import PMPParser
+#import pcmdi_metrics.driver.pmp_parser.PMPParser
```
Comment (doutriaux1) [inline src/python/monsoon_wang/scripts/mpindex_compute.py]: please remove

File: src/python/monsoon_wang/scripts/mpindex_compute.py [hunk 2]
```diff
@@ -16,14 +16,21 @@
 
 P.use("--modpath")
 P.use("--modnames")
-P.use("--results_dir")
+#P.use("--results_dir")
```
Comment (doutriaux1) [inline src/python/monsoon_wang/scripts/mpindex_compute.py]: cleanup please

File: src/python/monsoon_wang/scripts/mpindex_compute.py [hunk 3]
```diff
@@ -152,71 +111,19 @@
...
-        'MPI-ESM-MR',
-        'MPI-ESM-P',
-        'MRI-CGCM3',
-        'MRI-ESM1',
-        'NorESM1-M',
-        'NorESM1-ME']
+gmods = [] # TRAP LIST OF MODELS AVAILABLE WHICH MAY BE LESS THAN REQUESTED FROM PARAM FILE.
+for m in mods:
+  modpath.model = m
+  if os.path.isfile(modpath()) is True and m not in gmods:
```
Comment (doutriaux1) [inline src/python/monsoon_wang/scripts/mpindex_compute.py]: probably could make gmods a `set()` here would avoid checking if m already in it.

File: src/python/monsoon_wang/scripts/mpindex_compute.py [hunk 4]
```diff
@@ -286,6 +193,7 @@
         g.close()
     f.close()
 
+    print 'DONE WITH ', mod
```
Comment (doutriaux1) [inline src/python/monsoon_wang/scripts/mpindex_compute.py]: should we move to py3 print statements?

File: src/python/monsoon_wang/scripts/mpindex_compute.py [hunk 5]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
-
```
Comment (doutriaux1) [inline src/python/monsoon_wang/scripts/mpindex_compute.py]: let's move to py3 print statements and add
```
from __future__ import print_function
```

**Comment by gleckler1** [comment]
@doutriaux1 CD made your changes ... Note PG is recommending a changed here... all files in examples/parameters_files are to be "start-to-finish" working examples with data a user can download to verify.   Pls mv forward with this by either merging or making additional suggestion.

File: src/python/monsoon_wang/scripts/mpindex_compute.py (hunk 5 above)
Comment (gleckler1) [reply src/python/monsoon_wang/scripts/mpindex_compute.py]: @doutriaux1 ok but next time this line needs to be at top of code. ;-)

**Comment by doutriaux1** [comment]
@gleckler1 ok so I merge master in this one too. Apparently you made changes to wang parser but i did too, they're in two separate files right now, but a quick diff shows how much they differ. Do you remember what you changed so i can review it.

**Comment by doutriaux1** [comment]
@gleckler1 looking at the diff in the "file changed" section I'm pretty sure the changes were incorporated in a different PR. I'll further review next week.

**Comment by doutriaux1** [comment]
@gleckler1 what about all the deleted files?

**Pull request closed without merging**